.ddev/php/local-php.ini

```diff
@@ -7,3 +7,4 @@ memory_limit = 512M
 upload_max_filesize = 64M
 max_execution_time = 300
 max_input_time = 60
+apc.shm_size = 64M
```

Comment:
Careful with this one. This was only bumped up on DEPT yesterday as it is a large build with lots of PHP assets to cache. You can probably get away with the default on CORPLITE for now so I'd remove this until it's ever needed in future (you can monitor via the observability suite)